Add NavigatorWidget example

btheado opened the pull request:
Add an example using the NavigatorWidget and ListWidget to embed a storyriver inside a tiddler. A search box is provided for an easy way to get links to click. One hard-coded link is also provided when the storyriver is empty.

kookma commented:
@btheado - very nice and useful example! Thank you indeed!

There is one point, if you change the storyview to zoomin, the two line description below searchbox disappeared.
I think the issue is not with your solution, but with zoomin!

Thank you again!

Jermolene commented:
Thanks @btheado we already have the tiddler https://tiddlywiki.com/#Creating%20SubStories, perhaps we should improve that, and link to it from the navigator widget docs?

btheado commented:
@kookma, that is strange behavior. I think I'll hard-code the example to have storyview=classic. I think for the embedded case classic makes the most sense.

@Jermolene, I wasn't aware of that tiddler. I'll do as you suggest.

btheado commented:
I moved the example to the Creating Substories tiddler (replacing the example there). I hard-coded the storyview to classic. I also added more description and links.

Jermolene commented:
Thanks @btheado
